Sources/Views/GiftsV2.swift

```diff
@@ -0,0 +1,132 @@
+import Dependencies
+import StyleguideV2
+import TaggedMoney
+import Transcripts
+
+public struct GiftsV2: HTML {
```

Comment:
I'm using a lot from your `PricingV2` here, and I even moved a lot of the views from that page into the SharedComponents directory since they can be used directly in both screens.